turbinia/evidence.py

```diff
@@ -565,8 +552,14 @@ def _preprocess(self, _, required_states):
       if self.device_path:
         self.state[EvidenceState.ATTACHED] = True
         self.local_path = self.device_path
+    if EvidenceState.MOUNTED in required_states:
+      self.mount_path = mount_local.PreprocessMountPartition(self.device_path)
```

Comment:
Can you also set `self.local_path = self.mount_path` here just so the local_path always points to the evidence's last processed path?

Reply:
Done